101-rbac-builtinrole-resourcegroup/azuredeploy.json

```diff
@@ -2,45 +2,34 @@
   "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
   "contentVersion": "1.0.0.0",
   "parameters": {
-    "principalId": {
+    "roleAssignmentName": {
       "type": "string",
       "metadata": {
-        "description": "The principal to assign the role to"
+        "description": "Specifies the name of the role assignment to create. It can be any valid GUID."
       }      
     },
-    "builtInRoleType": {
```

Comment:
changing this behavior makes the readme inaccurate - need to reconcile that...

Reply:
Fixed.